### ISSUE TITLE
User avatars in notifications are sometimes links to my profile instead of theirs

### ISSUE THREAD

**Issue by carlwgeorge** [ronilaukkarinen/mastodon-bird-ui#25]
Using version 1.5.2, avatars in my notification list are sometimes links to my own profile instead of links to the profile of the user who performed the relevant action.

### favorites

[user_avatar]
[user_name] favourited your post

- [user_avatar] is a link to **my profile** (despite being the user's image)
- [user_name] is a link to the user's profile

### boosts

[user_avatar]
[user_name] boosted your post

- [user_avatar] is a link to **my profile** (despite being the user's image)
- [user_name] is a link to the user's profile

### follows

[user_avatar]
[user_name] followed you

- [user_avatar] is a link to the user's profile
- [user_name] is a link to the user's profile

**Comment by ronilaukkarinen** [comment]
Thanks. Something like `pointer-events: none;` for the hidden element should do. Will check it out when I have time. 👍

**Comment by ronilaukkarinen** [comment]
I just tested it and in the default UI avatar is linked to your own profile.

![image](https://user-images.githubusercontent.com/1534150/236669610-04aa8648-c507-4751-b874-6dc5e36e3b62.png)

In my UI I've flipped the logic around that the user avatar is from the one that actually does the action.

![image](https://user-images.githubusercontent.com/1534150/236670155-541aca6f-f975-4a71-a0d1-88477d344553.png)

It feels strange to have my own avatar in here. But the avatar is unfortunately not linked. I will just remove the pointer events from that so it doesn't confuse users.
